[0.021s] Invoking command in '/home/uditr/mybot_ws/build/mybot_msgs': CMAKE_PREFIX_PATH=/opt/ros/humble /usr/bin/cmake --build /home/uditr/mybot_ws/build/mybot_msgs -- -j8 -l8
[0.046s] -- Found ament_cmake: 1.3.9 (/opt/ros/humble/share/ament_cmake/cmake)
[0.195s] -- Found std_msgs: 4.2.4 (/opt/ros/humble/share/std_msgs/cmake)
[0.218s] -- Found rosidl_generator_c: 3.1.5 (/opt/ros/humble/share/rosidl_generator_c/cmake)
[0.220s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.226s] -- Found rosidl_generator_cpp: 3.1.5 (/opt/ros/humble/share/rosidl_generator_cpp/cmake)
[0.238s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.256s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.313s] -- Found geometry_msgs: 4.2.4 (/opt/ros/humble/share/geometry_msgs/cmake)
[0.358s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.535s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.535s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[0.535s]   mybot_msgs --arguments-file
[0.535s]   /home/uditr/mybot_ws/build/mybot_msgs/rosidl_adapter__arguments__mybot_msgs.json
[0.535s]   --output-dir
[0.536s]   /home/uditr/mybot_ws/build/mybot_msgs/rosidl_adapter/mybot_msgs
[0.536s]   --output-file
[0.536s]   /home/uditr/mybot_ws/build/mybot_msgs/rosidl_adapter/mybot_msgs.idls)
[0.536s]   returned error code 1:
[0.536s] 
[0.536s]   Traceback (most recent call last):
[0.536s] 
[0.536s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.536s]       return _run_code(code, main_globals, None,
[0.536s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.536s]       exec(code, run_globals)
[0.536s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.536s]       sys.exit(main())
[0.536s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.536s]       abs_idl_file = convert_to_idl(
[0.536s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 24, in convert_to_idl
[0.536s]       return convert_srv_to_idl(
[0.536s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/srv/__init__.py", line 28, in convert_srv_to_idl
[0.536s]       srv = parse_service_string(package_name, input_file.stem, content)
[0.536s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 850, in parse_service_string
[0.536s]       raise InvalidServiceSpecification(
[0.536s] 
[0.536s]   rosidl_adapter.parser.InvalidServiceSpecification: Could not find separator
[0.536s]   '---' between request and response
[0.536s] 
[0.536s] Call Stack (most recent call first):
[0.537s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.537s]   CMakeLists.txt:14 (rosidl_generate_interfaces)
[0.537s] 
[0.537s] [0m
[0.537s] -- Configuring incomplete, errors occurred!
[0.537s] See also "/home/uditr/mybot_ws/build/mybot_msgs/CMakeFiles/CMakeOutput.log".
[0.540s] gmake: *** [Makefile:747: cmake_check_build_system] Error 1
[0.542s] Invoked command in '/home/uditr/mybot_ws/build/mybot_msgs' returned '2': CMAKE_PREFIX_PATH=/opt/ros/humble /usr/bin/cmake --build /home/uditr/mybot_ws/build/mybot_msgs -- -j8 -l8
